packages/apps/spaces/.env.development

```diff
@@ -7,8 +7,8 @@ SETTINGS_API_KEY=8b010f38-e5ca-4923-a62e-9f073c5c7dbf
 FILE_STORAGE_API_PATH=http://localhost:10001
 FILE_STORAGE_API_KEY=9eb87aa2-75e7-45b2-a1e6-53ed297d0ba8
 
-OASIS_API_PATH=http://localhost:8006
-OASIS_API_KEY=10a6747a-97cd-4a6c-bcf5-e4ee89a12567
+COMMS_API_PATH=http://localhost:8013
+COMMS_API_KEY=10a6747a-97cd-4a6c-bcf5-e4ee89a12567
 
```

Comment:
could you add the following lines for webrtc?

NEXT_PUBLIC_TURN_SERVER=turn:127.0.0.1:3478?transport=tcp
NEXT_PUBLIC_TURN_USER=foo
NEXT_PUBLIC_TURN_CREDENTIAL=1234

I didn't add it to my branch as I wasn't touching spaces

Reply:
done